lib/app_theme.dart

```diff
@@ -1,10 +1,78 @@
 import 'package:flutter/material.dart';
 
-class AppTheme {
-  AppTheme._();
+class PrimaryAppTheme {
+  PrimaryAppTheme._();
+  static Color textfieldlabelColor(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.grey[300]
+        : Colors.grey[600];
+  }
+
+  static Color getTextColor(context) {
```

Comment:
better to name `textColor` for consistency among other function names involving returning colours based on theme!